src/dpos/dpos.ts

```diff
@@ -0,0 +1,319 @@
+import BN from 'bn.js'
```

Comment:
Can you rename this file to `dpos-user.ts`... seems that's the only thing in here.